app/src/js/components/home.js

```diff
@@ -147,6 +142,32 @@ class Home extends React.Component {
     }
   }
 
+  pageSection (className, title, children) {
+    return this.section({ className, title, children });
+  }
+
+  sectionHeader (className, title, link) {
+    return this.section({ className, title, link, size: 'large', border: true });
+  }
+
+  section ({ className, title, link, border = false, size = 'medium', children }) {
+    const borderMod = border ? '--border' : '';
+    const sizeMod = `--${size}`;
+    return (
+      <section className={`page__section ${className}`}>
+        <div className='row'>
+          <div className={`heading__wrapper${borderMod}`}>
+            <h2 className={`heading${sizeMod} heading--shared-content--right`}>
```

Comment:
Do we need the `heading--shared-content--right` class? Genuinely don't know, but was playing around with it in the browser and not sure that's it's doing much

Reply:
I can't really see anyplace it's used except the granule overview page and I can replace that with a sectionHeader now.

Reply:
Nope can't actually because the `row` div is causing problems, I don't want to rush though a bad change, so I'm going to leave just the homepage changes and leave some more refactoring for another time.